test/vtep/install.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#Please refer https://docs.cilium.io/en/stable/gettingstarted/kind/
+
+# 1 create kind cluster
+kind create cluster --config=kind-cluster.yaml
+
+# 2 install cilium
+
+helm repo add cilium https://helm.cilium.io/
+
+# docker pull the image and load in kind
+docker pull cilium/cilium:<cilium version>
+kind load docker-image cilium/cilium:<cilium version>
+
+helm install cilium cilium/cilium --version <cilium version> \
+   --namespace kube-system \
+   --set vtep.enabled=true \
+   --set vtep.endpoint="172.18.0.1" \
+   --set vtep.CIDR="10.1.5.0/24" \
+   --set vtep.MAC="00:50:56:A0:7D:D8" \
+   --set kubeProxyReplacement=partial \
+   --set hostServices.enabled=false \
+   --set externalIPs.enabled=true \
+   --set nodePort.enabled=true \
+   --set hostPort.enabled=true \
+   --set bpf.masquerade=false \
+   --set image.pullPolicy=IfNotPresent \
+   --set ipam.mode=kubernetes
+
+# 3 deploy busybox on kind control plaine node
+
+kubectl label node kind-control-plane  dedicated=master
+kubectl taint nodes --all node-role.kubernetes.io/master-
+
+kubectl apply -f busybox-master.yaml
+
+# 4 deploy vxlan-responder systemd service, note change
+#   change the bridge interface name in vxlan-responder.py
+#   to sniff
+
+cp vxlan-responder.service /etc/systemd/system/
```

Comment:
It might be nice to notify people that this will install a systemd service in their system:

```suggestion
echo "Enabling vxlan-responder.service by default in systemd..." >&2
cp vxlan-responder.service /etc/systemd/system/
```

Reply:
done